front/lib/api/assistant/visualization.ts

```diff
@@ -108,7 +108,6 @@ Guidelines using the :::visualization tag:
 - Props:
   - The generated component should not have any required props / parameters
 - Responsiveness:
-  - The content should be responsive and should not have fixed widths or heights
```

Comment:
So merely removing this line is enough for the model to always pick an appropriate height ?

Reply:
Maybe we might want to guide it a bit with a range of sizes it can pick from ? Small (200px), Medium (400px), Large (600px) and extra large (800px) ? Taller than that it starts to be too much I think.

Reply:
Agreed, trying this out

Reply:
My understanding of it is that the model outputs charts with an aspect ratio that makes sense

Reply:
indeed, but the model can't really know how much width is available (and we don't want any horizontal scroll bars either)